packages/docs/site/docs/10-javascript-api/03-playground-api-client.md

```diff
@@ -70,6 +70,45 @@ await client.goTo('/test/index.php');
 
 For a full list of these methods, consult the PlaygroundClient interface.
 
+## Sending messages to JavaScript
+
+You can pass messages from PHP to JavaScript using the `post_message_to_js()` function. It accepts two arguments:
+
+-   `$type` – is used to call the correct listener
+-   `$data` – any extra information as a string
+
+For example, here's how you would send a message called "post-save" with a JSON-encoded post ID and title:
+
+```ts
+const php = await PHP.load('8.0');
+
+php.onMessage(
```

Comment:
note that JS APIs use `.onmessage` without camel-casing the event name. we may not want to make _this_ `postMessage()` different than a normal one in that regard, but it's allowed

Reply:
🤔 maybe an entirely different name would better here since this isn't passing an `Event` object but a `data` string

Reply:
Oh, JS also does this:

```js
window.onmessage = function() {}
```

Whereas this API does:

```js
php.onMessage(function(){})
```

Perhaps `onPHPMessage` would be distinct enough, or even `onPHPEvent`. The prefix `on` is already established in a few other methods so it's mostly a matter of what comes after it.